.version-state.nix

```diff
@@ -0,0 +1,4 @@
+# See nixos/doc/manual/development/releases.xml, subsection "After the final release time"
+{
+  state = "supported";
+}
```

Comment:
Oh and this file should probably be in the nixos folder instead

Reply:
Should it be in the `nixos` folder, if it also affects the Darwin channels and stable channels used outside NixOS? (Even though this PR doesn't add anything for those right now)